assets/js/base/components/payment-methods/payment-method-tab.js

```diff
@@ -36,7 +38,7 @@ const PaymentMethodTab = ( { children, allowsSaving } ) => {
 	return (
 		<PaymentMethodErrorBoundary isEditor={ isEditor }>
 			{ children }
-			{ customerId > 0 && allowsSaving && (
+			{ customerId > 0 && allowsSaving && ! requiresSaving && (
```

Comment:
hmm this conditional seems a bit wonky to me. Don't we want something like:
```suggestion
			{ customerId > 0 && ( allowsSaving || requiresSaving ) && (
```

In other words, "If a customer is logged in, and the payment method allows saving or requires saving, then show the checkbox".